libs/community/langchain_community/vectorstores/azure_cosmos_db.py

```diff
@@ -397,8 +418,10 @@ def _similarity_search_with_score(
         embeddings: List[float],
         k: int = 4,
         kind: CosmosDBVectorSearchType = CosmosDBVectorSearchType.VECTOR_IVF,
+        pre_filter: Optional[Dict] = None,
         ef_search: int = 40,
         score_threshold: float = 0.0,
+        with_embedding: bool = False,
```

Comment:
Can you add new args to docstring

Reply:
@isahers1  What do you mean by docstring?